into/backends/spark.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import division, print_function, absolute_import
+
+
+class Dummy(object):
+    pass
+
+try:
+    from pyspark import SparkContext
+    import pyspark
+    from pyspark import RDD
+    from pyspark.rdd import PipelinedRDD
+    from pyspark.sql import SchemaRDD, SQLContext
+    RDD.min
+except (AttributeError, ImportError):
+    SchemaRDD = PipelinedRDD = RDD = SparkContext = SQLContext = Dummy
+    pyspark = Dummy()
+
+
+from datashape import var
+
+from .. import convert, append, discover
+
+
+@append.register(SparkContext, list)
+def list_to_spark_context(sc, seq, **kwargs):
+    return sc.parallelize(seq)
+
+
+@append.register((SparkContext, SQLContext), object)
```

Comment:
Sorry, meant `SQLContext`.  If I try to convert `into(mySQLContext, my_pandas_dataframe)` then I would expect this to go through `DataFrame -> list -> RDD -> SchemaRDD`.  I would expect the last hop to be from an `RDD` into a `SQLContext`.  This is also the `append` implementation that you have below.  

Ah, now that I see it you have two different `append` functions.  When would I prefer one over the other, e.g. what does `list -> SchemaRDD` have over `list -> RDD -> SchemaRDD`?